src/main/java/lotto/controller/LottoGameController.java

```diff
@@ -4,10 +4,15 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
+import java.util.List;
+
 public class LottoGameController {
     public void start() {
         Money money = InputView.inputMoney();
-        LottoGame lottoGame = LottoStore.sell(money);
+        int manualTryCount = InputView.inputManualTryCount();
+        List<String> manualNumbersStrings = InputView.inputManualNumbers(manualTryCount);
+
+        LottoGame lottoGame = LottoStore.sell(manualNumbersStrings,money);
```

Comment:
사소하지만 컨벤션 관련해서 습관적으로 체크해주시는것도 도움이 될 것 같습니다 👍 
인텔리제이 + mac 을 사용하신다면 option + command + L 로 전체 소스코드를 정리할 수 있습니다 ~

```suggestion
        LottoGame lottoGame = LottoStore.sell(manualNumbersStrings, money);
```

Reply:
감사합니다!